<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseParametersControlWidget</class>
 <widget class="QWidget" name="BaseParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Base Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="flowInputBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="6" column="8">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="5" colspan="2">
       <widget class="QCheckBox" name="mirrorCheckBox">
        <property name="text">
         <string>Mirror</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QComboBox" name="rotationComboBox">
        <item>
         <property name="text">
          <string>No rotation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90deg rotation</string>
         </property>
         <property name="icon">
          <iconset resource="../../../resources/main.qrc">
           <normaloff>:/new/prefix1/transform_rotate_90.png</normaloff>:/new/prefix1/transform_rotate_90.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180deg rotation</string>
         </property>
         <property name="icon">
          <iconset resource="../../../resources/main.qrc">
           <normaloff>:/new/prefix1/transform_rotate_180.png</normaloff>:/new/prefix1/transform_rotate_180.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>270deg rotation</string>
         </property>
         <property name="icon">
          <iconset resource="../../../resources/main.qrc">
           <normaloff>:/new/prefix1/transform_rotate_270.png</normaloff>:/new/prefix1/transform_rotate_270.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QComboBox" name="interpolationTypeComboBox">
        <item>
         <property name="text">
          <string>Nearest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bilinear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bilinear Fixed8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hardware</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Canvas</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QSpinBox" name="hSpinBox">
        <property name="toolTip">
         <string>height</string>
        </property>
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>2024</number>
        </property>
        <property name="value">
         <number>640</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QSpinBox" name="wSpinBox">
        <property name="toolTip">
         <string>width</string>
        </property>
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>4048</number>
        </property>
        <property name="value">
         <number>480</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QCheckBox" name="autoHCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="downsampleSpinBox">
        <property name="prefix">
         <string>/</string>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>16.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QCheckBox" name="autoWCheckBox">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="loadButton">
        <property name="toolTip">
         <string>Load Transform</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/new/prefix1/remove.png</normaloff>:/new/prefix1/remove.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QSpinBox" name="ySpinBox">
        <property name="toolTip">
         <string>y</string>
        </property>
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>-2048</number>
        </property>
        <property name="maximum">
         <number>2048</number>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QSpinBox" name="xSpinBox">
        <property name="toolTip">
         <string>x</string>
        </property>
        <property name="suffix">
         <string>px</string>
        </property>
        <property name="minimum">
         <number>-1024</number>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="5">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="filter2Button">
           <property name="text">
            <string>R</string>
           </property>
           <property name="icon">
            <iconset resource="../../../resources/main.qrc">
             <normaloff>:/new/prefix1/filter.png</normaloff>:/new/prefix1/filter.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="filterLockButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../resources/main.qrc">
             <normaloff>:/new/prefix1/unlock.png</normaloff>
             <normalon>:/new/prefix1/lock.png</normalon>:/new/prefix1/unlock.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="filterButton">
           <property name="text">
            <string>L</string>
           </property>
           <property name="icon">
            <iconset resource="../../../resources/main.qrc">
             <normaloff>:/new/prefix1/filter.png</normaloff>:/new/prefix1/filter.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="swapCamerasButton">
        <property name="toolTip">
         <string>Swap Camera Inputs. Exchange Left and Right</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/new/prefix1/exchange.png</normaloff>:/new/prefix1/exchange.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="filterGraphButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../resources/main.qrc">
          <normaloff>:/new/prefix1/brick_add.png</normaloff>:/new/prefix1/brick_add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="enableFilterGraphCheckBox">
        <property name="text">
         <string>Enable Filter Graph</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
  <include location="../../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>downsampleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>interpolationTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mirrorCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotationComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableFilterGraphCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>BaseParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
</ui>
